apps/aehyperchains/src/aehc_connector.erl

```diff
@@ -0,0 +1,98 @@
+-module(aehc_connector).
+
+-export([send_tx/1, get_block/1]).
+
+-export([tx/3, header/2, block/2]).
+
+-export([publish/2, subscribe/1]).
+
+-type connector() :: atom().
+
+-callback send_tx(Tx::aetx:tx()) -> binary().
+
+-callback get_block(Num::integer()) -> block().
+
+-export_type([tx/0, header/0, block/0]).
+
+-spec connector() -> connector().
```

Comment:
What is the purpose of it?

Reply:
To emphasize some kind of protocol related stuff (maybe you are right, looks weird, sorry just habit from previous work)